robottelo/ssh.py

```diff
@@ -241,6 +241,22 @@ def upload_file(local_file, remote_file, key_filename=None, hostname=None):
             sftp.close()
 
 
+def upload_files(local_dir, remote_dir, file_type='.txt'):
+    """ Upload all files from directory to a remote directory
+    :param local_dir: all files from local path to be uploaded.
+    :param remote_dir: a remote path where the uploaded files will be
+           placed.
+    :param file_type: filter only files contains the type extension
+    """
+    command("mkdir -p {}".format(remote_dir))
+    for root, dirs, files in os.walk(local_dir):
+        for local_filename in files:
+            if file_type in local_filename:
```

Comment:
Did you intend to match like this? i.e. also match `bla.txt.gz` or did you intend `os.path.splitext(local_filename)[1] == file_type`?

Reply:
make sense, will do that change